protocol/protobuf/communities.proto

```diff
@@ -88,6 +93,7 @@ message CommunityDescription {
   repeated string tags = 14;
   map<string, CommunityTokenPermission> token_permissions = 15;
   repeated CommunityTokenMetadata community_tokens_metadata = 16;
+  map<string, MemberWallet> member_wallets = 17;
```

Comment:
Realized that we don't necessarily need a new `MemberWallets` field, could also leverage `CommunityMember` instead. Will update this.